package.json

```diff
@@ -55,22 +50,13 @@
 	},
 	"funding": "https://github.com/sponsors/sergiodxa",
 	"homepage": "https://github.com/sergiodxa/remix-i18next#readme",
-	"keywords": [
-		"remix",
-		"i18n",
-		"i18next",
-		"ssr",
-		"csr"
-	],
+	"keywords": ["remix", "i18n", "i18next", "ssr", "csr"],
 	"license": "MIT",
 	"peerDependenciesMeta": {
-		"@remix-run/cloudflare": {
+		"@react-router/cloudflare": {
 			"optional": true
 		},
-		"@remix-run/deno": {
-			"optional": true
-		},
```

Comment:
deno got removed from rr7 from what I could see

Reply:
These are not needed anymore, as RR doesn't have a server-runtime package, so remix-i18next imports directly from react-router, this eliminates any dependency on the /cloudflare or /node packages

Reply:
Tbh I wasn't sure why you were using the node one so I just left that one in just in case but yeah, you can also try a demo by running:
npm i https://pkg.pr.new/AlemTuzlak/remix-i18next@047de17

Reply:
I just tested it out on our internal react-router v7 stack and it works without issues!

Reply:
We only really needed the `/server-runtime` for the types, but because that's an internal thing I set the peer dependencies to the wrappers `/node`, `/cloudflare` and `/deno`, so you also installed the `/server-runtime` with that and without having to force you to include `/server-runtime` in your package.json

Now that the types and implementations for Cookie and SessionStorage comes both from react-router there's no reason to use the runtime packages anymore here, this is finally fully independent of the server runtime.